manipulator_1:
  regions:
    pick_region:
      min_limits:
        [-0.5, 0.4, 0.90, 0, 0, 0] #1.570796
      max_limits:
        [0.4, 0.9, 0.90, 0, 0, 0]
    place_region:
      min_limits:
        [0.7, -0.25, 0.75, 0.0, 0.0, 0.0]
      max_limits:
        [1.2, 0.25, 0.75, 0.0, 0.0, 0.0]

manipulator_2:
  regions:
    pick_region:
      min_limits:
        [ 1.60, -1.24, 0.72 , 0.0, 0.0, 0.0 ]
      max_limits:
        [ 1.90, -0.74, 0.72, 0.0, 0.0, 0.0 ]
    place_region:
      min_limits:
        [0.7, -0.25, 0.7, 0.0, 0.0, 1.570796]
      max_limits:
        [1.2, 0.25, 0.7, 0.0, 0.0, 1.570796]

manipulator_3:
  regions:
    pick_region:
      min_limits:
        [ 1.60, 0.74, 0.86 , 0.0, 0.0, 0.0]
      max_limits:
        [ 1.90, 1.24, 0.86, 0.0, 0.0, 0.0 ]
    place_region:
      min_limits:
        [0.7, -0.25, 0.75, 0.0, 0.0, 1.570796]
      max_limits:
        [1.2, 0.25, 0.75, 0.0, 0.0, 1.570796]